azext_iot/tests/test_iot_central_int.py

```diff
@@ -213,7 +213,7 @@ def test_central_device_registration_info_filter_registered(self):
             device_info_results.append(device.get("device_info"))
 
         for device in device_info_results:
-            assert device.get("deviceStatus") == device_status_expected
+            assert device.get("device_status") == device_status_expected
```

Comment:
I'd say add UT that will check the device_info portion is restricted to the subset of information that we get back from `get_device_registration_info`